google-cloud-video-stitcher/samples/create_vod_config.rb

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START videostitcher_create_vod_config]
+require "google/cloud/video/stitcher"
+
+##
+# Create a VOD config. VOD configs are used to create VOD sessions.
+#
+# @param project_id [String] Your Google Cloud project (e.g. "my-project")
+# @param location [String] The location (e.g. "us-central1")
+# @param vod_config_id [String] Your VOD config name (e.g. "my-vod-config")
+# @param source_uri [String] Uri of the VOD stream to stitch
+#   (e.g. "https://storage.googleapis.com/my-bucket/main.mpd")
```

Comment:
Please enclose these examples in backticks instead of double quotes, especially if they're URLs or look like URLs (since documentation generators might try to linkify URLs that aren't being treated as code-font strings). Also check any similar cases in your other new samples.

Reply:
Done